src/component/container/LayerTreeClassic/LayerTreeClassic.tsx

```diff
@@ -72,7 +78,28 @@ export class LayerTreeClassic extends React.Component<LayerTreeClassicProps> {
     } else {
       return (
         <div>
-          {layer.get('name')}
+          <div className="classic-tree-node-header">
+            <div>
+              {layer.get('name')}
+            </div>
+            {showLayerButtons &&
```

Comment:
The prop `showLayerButtons` is very unspecific. Please add a prop for each tree button, e.g. `showApplyTimeInterval` and `showContextMenu`.

Reply:
Right!